content/en/blog/_posts/2022-12-06-kubernetes-1.25-blog.md

```diff
@@ -0,0 +1,190 @@
+
+It's with immense joy that we announce the release of Kubernetes v1.26!
+
+This release includes a total of 37 enhancements: eleven of them are graduating to Stable, ten are graduating to Beta, and sixteen of them are entering Alpha. We also have two features being deprecated or removed.
+
+## Release theme and logo 
+
+**Kubernetes 1.26: Electrifying**
+
+\<image here>
+
+The theme for Kubernetes v1.26 is _Electrifying_.
+
+
+\<explanation of the theme>
+
+## Major Themes
+    
+Kubernetes v1.26 is composed of many changes, brought to you by a worldwide team of volunteers. For this release, we have identified several major themes.
+
+### Progress on the CSI migration
+
+Following the GA of the [core Container Storage Interface (CSI) Migration](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/625-csi-migration) feature in the previous release, CSI migration is an on-going effort that we've been working on for a few releases now, and this release continues to add (and remove) features aligned with the migration's goals.
+
+#### Delegate FSGroup to CSI Driver graduated to Stable
+
+This feature allows Kubernetes to [supply the pod's `FSGroup` to the CSI driver when a volume is mounted](https://github.com/kubernetes/enhancements/issues/2317) so that the driver can utilize mount options to control volume permissions. Previously, the kubelet would always apply the `fsGroup`ownership and permission change to files in the volume according to the policy specified  in the `Pod.spec.securityContext.fsGroupChangePolicy` field. Starting with this release, CSI drivers have the option to apply the `fsGroup` settings during attach or mount time of the volumes.
+
+#### CSI migration for Azure File and vSphere graduated to Stable
+
+Both the [vSphere](https://github.com/kubernetes/enhancements/issues/1491) and [Azure](https://github.com/kubernetes/enhancements/issues/1885) in-tree driver migration to CSI have graduated to Stable.
+
+#### In-tree GlusterFS driver removal
+
+Already deprecated in the v1.25 release, the [in-tree GlusterFS driver](https://github.com/kubernetes/enhancements/issues/3446) was removed in this release.
+
+#### In-tree OpenStack Cinder driver removal
+
+This release removed the deprecated in-tree storage integration for OpenStack (the `cinder` volume type). You should migrate to external cloud provider and CSI driver from https://github.com/kubernetes/cloud-provider-openstack instead. For more information, visit [Cinder in-tree to CSI driver migration](https://github.com/kubernetes/enhancements/issues/1489).
+
+### CRI v1alpha2 removed
+
+With the adoption of the [Container Runtime Interface](https://kubernetes.io/docs/concepts/architecture/cri/) (CRI) and the [removal of dockershim] in v1.24 , the CRI is the supported and documented way through which Kubernetes interacts with different container runtimes. Each kubelet negotiates which version of CRI to use with the container runtime on that node.
```

Comment:
Is `removal of dockershim` a broken hyperlink?

Reply:
It was, fixed.